packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -52,52 +59,12 @@ class _FrameCallbackEntry {
   StackTrace stack;
 }
 
-class Priority {
-  static const Priority idle = const Priority._(0);
-  static const Priority animation = const Priority._(100000);
-  static const Priority touch = const Priority._(200000);
-
-  /// Relative priorities are clamped by this offset.
-  ///
-  /// It is still possible to have priorities that are offset by more than this
-  /// amount by repeatedly taking relative offsets, but that's generally
-  /// discouraged.
-  static const int kMaxOffset = 10000;
-
-  const Priority._(this._value);
-
-  int get value => _value;
-  final int _value;
-
-  /// Returns a priority relative to this priority.
-  ///
-  /// A positive [offset] indicates a higher priority.
-  ///
-  /// The parameter [offset] is clamped to +/-[kMaxOffset].
-  Priority operator +(int offset) {
-    if (offset.abs() > kMaxOffset) {
-      // Clamp the input offset.
-      offset = kMaxOffset * offset.sign;
-    }
-    return new Priority._(_value + offset);
-  }
-
-  /// Returns a priority relative to this priority.
-  ///
-  /// A positive offset indicates a lower priority.
-  ///
-  /// The parameter [offset] is clamped to +/-[kMaxOffset].
-  Priority operator -(int offset) => this + (-offset);
-}
-
-/// Scheduler running tasks with specific priorities.
+/// Scheduler for running scheduled frames at system-controlled
+/// intervals and tasks according to their priorities during the
+/// time between frames.
 ///
```

Comment:
The new version of the comment seems to run on a little.  Maybe:

The Scheduler runs frames at system-controlled intervals and it runs tasks during the time between frames.  Tasks are run in priority order.